api/src/integration/java/org/pmiops/workbench/google/DirectoryServiceImplIntegrationTest.java

```diff
@@ -1,7 +1,6 @@
 package org.pmiops.workbench.google;
 
 import static com.google.common.truth.Truth.assertThat;
-import static com.google.common.truth.Truth8.assertThat;
```

Comment:
this is deprecated and redundant: all of it is now included in the base `Truth`.